src/components/forms/SignatureField/Signature.scss

```
@@ -0,0 +1,25 @@
+@use '@utrecht/components/form-field' as form-field;
+@use '@/scss/form-field-override';
+
+@use '@/scss/bem';
+
+.openforms-signature-footer {
+  padding: 0.1em;
+  text-align: center;
+  color: var(--of-color-fg-muted);
```

```suggestion
  color: var(--of-signature-footer-color, var(--of-color-fg-muted));
```
proper design token structure - a component should only consume it's own design tokens and be highly specific to the component itself.

The fallback value is for backwards compatibility.